apps/src/StudioApp.js

```diff
@@ -146,8 +146,8 @@ var StudioAppClass = function () {
 
   this.MIN_WORKSPACE_HEIGHT = undefined;
 };
-
-module.exports = StudioAppClass;
+var studioAppSingleton = new StudioAppClass();
+module.exports = studioAppSingleton;
```

Comment:
1. Sure.  Actually tried this at one point and switched when I noticed `StudioAppClass` wasn't used directly anywhere.  Shall I switch back to using a `.singleton` "static" member?
2. Why not :+1: 
3. Makes sense, but I think I'd rather change (1) and (2) and leave it named for the class.